target/linux/ramips/image/lzma-loader/src/Makefile

```diff
@@ -54,7 +53,27 @@ LDFLAGS		+= -flto -fwhole-program -Wl,-z,max-page-size=4096
 
 O_FORMAT 	= $(shell $(OBJDUMP) -i | head -2 | grep elf32)
 
-OBJECTS		:= head.o loader.o cache.o board-$(PLATFORM).o printf.o LzmaDecode.o
+OBJECTS		:= head.o loader.o cache.o board.o printf.o LzmaDecode.o
+
+ifeq ($(SUBTARGET),)
+$(error "Please specify a SUBTARGET!")
+endif
+
+ifeq ($(strip $(SUBTARGET)),mt7620)
+CFLAGS	+= -DSOC_MT7620
+endif
+
+ifeq ($(SUBTARGET),mt7621)
+CFLAGS	+= -DSOC_MT621
```

Comment:
Should be `-DSOC_MT7621`

Reply:
Fixed, I have just realized that CI only builds the kernel. Will compile all the targets and will report back if something fails.

Thanks!

Reply:
Should I resolve this comments or should @Spudz76? What is your usual way to proceed? Thanks!

Reply:
I can't mark resolved because I don't have that button.